modules/core/arrow-test/src/main/kotlin/arrow/test/laws/Law.kt

```diff
@@ -5,9 +5,12 @@ import arrow.test.generators.tuple3
 import arrow.test.generators.tuple4
 import arrow.test.generators.tuple5
 import arrow.typeclasses.Eq
+import io.kotlintest.Matcher
+import io.kotlintest.Result
 import io.kotlintest.TestContext
 import io.kotlintest.properties.Gen
 import io.kotlintest.should
+import io.kotlintest.shouldBe
```

Comment:
```suggestion
```

Reply:
This is not silly at all, I should have looked more closely!